upgrade/build.gradle

```diff
@@ -0,0 +1,53 @@
+import dependencies.Dependencies
+import dependencies.Versions
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'kotlin-kapt'
+}
+
+apply from: '../config/quality.gradle'
+
+android {
+    compileSdkVersion Versions.android_compile_sdk
+
+    defaultConfig {
+        minSdkVersion Versions.android_min_sdk
+        targetSdkVersion Versions.android_target_sdk
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
+}
+
+dependencies {
+    implementation Dependencies.androidx_core_ktx
+    implementation project(path: ':shared')
```

Comment:
You can skip path: we use both versions it would be good to choose one and use across the app.